### DISCUSSION TITLE
Please upload a 1.23.0 Noble release for Linux

### DISCUSSION THREAD

**Discussion by vtnate** [OpenModelica/OpenModelica#12661] [Q&A] [answered]
I'm interested in testing OM 1.23 on Ubuntu 24 in my docker container, but it's not available yet at https://build.openmodelica.org/omc/builds/linux/releases/1.23.0/dists/. I'm duplicating [your dockerfile](https://github.com/OpenModelica/OpenModelicaDockerImages/blob/v1.23.0/Dockerfile) and experimenting with the new version of Ubuntu.

Thank you!

**Comment by casella** [comment]
@AnHeuermann, @sjoelund what is the status with that?

**Reply by AnHeuermann** [reply, marked as answer]
@adrpo added https://build.openmodelica.org/omc/builds/linux/releases/1.23.0/dists/noble/ (and fixed the script for the future)

**Comment by AnHeuermann** [comment]
OpenModelica stable branch has v1.23.0 available, but it's not copied into the releses directory yet.

You can install omc inside a docker container via:

```dockerfile
FROM ubuntu:noble

ENV SHELL /bin/bash

# Ensure DEBIAN_FRONTEND is only set during build
ARG DEBIAN_FRONTEND=noninteractive

RUN apt-get update && apt-get upgrade -qy && apt-get dist-upgrade -qy
RUN apt-get install -qy \
  ca-certificates       \
  curl                  \
  gnupg                 \
  lsb-release

# Install build-deps of OpenModelica
RUN curl -fsSL http://build.openmodelica.org/apt/openmodelica.asc | gpg --dearmor -o /usr/share/keyrings/openmodelica-keyring.gpg
RUN echo \
  "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/openmodelica-keyring.gpg] https://build.openmodelica.org/apt \
  $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/openmodelica.list > /dev/null
RUN apt-get update && apt-get install -qy omc cmake

# Clean
RUN rm -rf /var/lib/apt/lists/* \
  && apt-get clean
```

```bash
$ docker build --tag=omc-noble . 
$ docker run --rm -it omc-noble omc --version
OpenModelica 1.23.0~3-g4080a7b
```

The version tag isn't correct, but it is indeed the stable relesae.
